types/operator.go

```diff
@@ -83,15 +83,16 @@ type StakeAmount = *big.Int
 // It is currently the hash of the operator's G1 pubkey (in the bls pubkey registry)
 type OperatorId = [32]byte
 type QuorumNum = uint8
-type BlockNumber = uint32
+type QuorumThresholdPercentage = uint32
+type BlockNum = uint32
```

Comment:
Why can't we keep the type `BlockNumber` ?

Reply:
Changed it to `BlockNum` so it fits with `QuorumNum`